manage.py

```diff
@@ -115,25 +113,13 @@ def run_patches():
     import mimetypes
     mimetypes.init()
 
-    patch_pickle()
     patch_jsonfield()
 
     import django
     _setup_once.setup = django.setup
     django.setup = _setup_once
 
 
-def patch_pickle():
-    """Patch pickle to support protocol 5
-
-    Remove after upgrading to Python 3.8+
-    """
-    import pickle
-    if pickle.HIGHEST_PROTOCOL < 5:
```

Comment:
Will we need this function again the next time a python upgrade includes a new highest pickle protocol?

Reply:
If we do I know where to dig it up. I hope we are upgraded to a new version of Celery with better behavior by the time we would need it.